recipes/ffmpeg/all/conanfile.py

```diff
@@ -537,55 +535,61 @@ def opt_append_disable_if_set(args, what, v):
         if not self.options.with_programs:
             args.append("--disable-programs")
         # since ffmpeg"s build system ignores CC and CXX
-        if tools.get_env("AS"):
-            args.append("--as={}".format(tools.get_env("AS")))
-        if tools.get_env("CC"):
-            args.append("--cc={}".format(tools.get_env("CC")))
-        if tools.get_env("CXX"):
-            args.append("--cxx={}".format(tools.get_env("CXX")))
+        as_value = os.getenv("AS")
+        if as_value:
+            args.append(f"--as={as_value}")
+        cc_value = os.getenv("CC")
+        if cc_value:
+            args.append(f"--cc={cc_value}")
+        cxx = os.getenv("CXX")
+        if cxx:
+            args.append(f"--cxx={cxx}")
         extra_cflags = []
         extra_ldflags = []
         if is_apple_os(self) and self.settings.os.version:
-            extra_cflags.append(tools.apple_deployment_target_flag(
+            extra_cflags.append(apple_deployment_target_flag(
                 self.settings.os, self.settings.os.version))
-            extra_ldflags.append(tools.apple_deployment_target_flag(
+            extra_ldflags.append(apple_deployment_target_flag(
                 self.settings.os, self.settings.os.version))
-        if self._is_msvc:
-            args.append("--pkg-config={}".format(tools.get_env("PKG_CONFIG")))
+        if is_msvc(self):
+            pkg_config = os.getenv("PKG_CONFIG")
+            args.append(f"--pkg-config={pkg_config}")
             args.append("--toolchain=msvc")
-            if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) <= "12":
+            if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) <= "12":
                 # Visual Studio 2013 (and earlier) doesn't support "inline" keyword for C (only for C++)
                 self._autotools.defines.append("inline=__inline")
         if cross_building(self):
             if self._target_os == "emscripten":
                 args.append("--target-os=none")
             else:
-                args.append("--target-os={}".format(self._target_os))
+                args.append(f"--target-os={self._target_os}")
```

Comment:
```suggestion
                target_os = "Macos" if is_apple_os(self) else str(self.settings.os)
                args.append(f"--target-os={target_os}")
```
So you can get rid of the other `_target_os` or at least the `_get_gnu_triplet`